projects/common/src/constants/application-constants.ts

```diff
@@ -1,4 +1,5 @@
 export const enum ApplicationFeature {
   PageTimeRange = 'ui.page-time-range',
-  FeatureDefaultTimeRangeMap = 'ui.feature-default-time-range-map'
+  FeatureDefaultTimeRangeMap = 'ui.feature-default-time-range-map',
+  DisableIncrementalSearch = 'ui.disable-incremental-search'
```

Comment:
Make feature flags consistent, it should be enabling only. Add a true default for backwards compat.

Reply:
updated.